http4k-server/ktorcio/src/test/kotlin/org/http4k/server/KtorCIOTest.kt

```diff
@@ -27,3 +27,12 @@ class KtorCIOTest : ServerContract({ KtorCIO(Random().nextInt(1000) + 8745) }, A
 
     override fun requestScheme(): Matcher<String?> = equalTo("http")
 }
+
+class KtorCIOStopTest : ServerStopContract(
+    { stopMode -> KtorCIO(Random().nextInt(1000) + 8745, stopMode) },
+    ApacheClient(),
+    {
+        enableImmediateStop()
+        enableGracefulStop()
+        enableDelayedStop()
```

Comment:
Not sure if `Delayed` stop should be added when `Graceful` already works as `Delayed` should only be for cases where `Graceful` is not possible